core/pom.xml

```diff
@@ -24,7 +24,11 @@
       <artifactId>security-spring</artifactId>
       <version>${project.version}</version>
     </dependency>
-
+      <dependency>
```

Comment:
Making a dependency on persistence-spark so we can place the parquet writer here doesn't sit right with me, but it is consistent with where the other scripts go.  Should we consider keeping it in persistence-spark until a better place is identified?